### ISSUE TITLE
Improve build of NCEPlibs

### ISSUE THREAD

**Issue by mmiesch** [JCSDA/jedi-stack#43]
Currently we install NCEPlibs from a subdirectory in jedi-stack because its convenient.  But, we should really download each library from the github site: https://github.com/NOAA-EMC and then configure each one as described there.  But, I'm not sure it's worth the trouble now because (1) JEDI doesn't currently need them and (2) EMC is working on a new CMake build system for them.  Still, I'm putting it in the icebox as something that should be done eventually.

**Comment by markjolah** [comment]
Agreed this should not be a priority until someone actually is using these with JEDI.  When it comes time to do it, I designed the build system of [JCSDA/bufrlib](https://github.com/JCSDA/bufrlib/) so that it could nearly be a copy and paste for some of these other libraries.  I'd like to confirm we have all the bugs worked out of the bufrlib CMake for intel/HPC etc.  I think the project with @srherbener  and @hu5970 to design GSI prep bufr converters will sufficiently test the bufrlib build system once we have it working with MPI etc on HPC platforms.

**Comment by aerorahul** [comment]
@mmiesch Here is a Cmake build of some nceplibs from GMAO. https://github.com/GEOS-ESM/NCEP_Shared?files=1
